gt/main.c

```diff
@@ -762,9 +765,13 @@ gt_proc(void *arg)
 			 */
 			ret = gt_parse_incoming_pkt(m, &pkt_info);
 			if (ret < 0) {
-				if (pkt_info.outer_ip_ver == ETHER_TYPE_IPv6) {
+				switch (pkt_info.outer_ethertype) {
+				case ETHER_TYPE_IPv6:
 					add_pkt_ipv6_acl(&acl, m);
 					continue;
+				case ETHER_TYPE_ARP:
```

Comment:
Somehow, this patch and the previous patch swap order. And this part of the code has a conflict.